book/src/frameworks.md

```diff
@@ -0,0 +1,170 @@
+# Rinja in web frame
+
+Rinja's [`Template::render()`] returns <code>Result&lt;String, [rinja::Error]&gt;</code>.
+To make this result work in your preferred web-framework, you'll need to handle both cases:
+converting the `String` to a web-response with the correct `Content-Type`,
+and the `Error` case to a proper error message.
+
+While in many cases it will be enough to simply convert the `Error` to
+
+* `Box<dyn std::error::Error + Send + Sync>` using `err.into_box()` or
+* `std::io::Error` using `err.into_io_error()`
+
+it is **recommended** to use a custom error type.
+This way you can display the error message in your app's layout,
+and you are better prepared for the likely case that your app grows in the future.
+Maybe you'll need to access a database and handle errors?
+Maybe you'll add multiple languages and you want to localize error messages?
+
+The crates [`thiserror`] and [`displaydoc`] can be useful to implement this error type.
+
+[`Template::render()`]: <https://docs.rs/rinja/0.3.5/rinja/trait.Template.html#method.render>
+[rinja::Error]: <https://docs.rs/rinja/0.3.5/rinja/enum.Error.html>
+[`thiserror`]: <https://crates.io/crates/thiserror>
+[`displaydoc`]: <https://crates.io/crates/displaydoc>
+
+## Actix-Web and Rinja
+
+To convert the `String` to an HTML response, you can use
+[`Html::new(_)`](https://docs.rs/actix-web/4.9.0/actix_web/web/struct.Html.html#method.new).
+
+```rust
+use actix_web::Responder;
+use actix_web::web::Html;
+
+fn handler() -> Result<impl Responder, AppError> {
+    …
```

Comment:
```suggestion
    // …
```

I prefer for the syntax to be fully rust compatible but this is just a nit so feel free to ignore it.

Reply:
I intentionally made it a syntax error, so you can't ignore it. :D

Reply:
Ah I see. Well that works for me! :laughing: